types/svg-parser/index.d.ts

```diff
@@ -1,14 +1,24 @@
-// Type definitions for svg-parser 1.0
+// Type definitions for svg-parser 2.0
 // Project: https://gitlab.com/Rich-Harris/svg-parser#README
```

Comment:
The `svg-parser@2.x.x` now lives on GitHub.

https://github.com/Rich-Harris/svg-parser

Reply:
I hadn't even checked!  Thanks for that!